src/content/community/grants/index.md

```diff
@@ -29,7 +29,7 @@ These projects have created their own grants for projects aimed at developing an
 
```

Comment:
I didn't ran yarn format, its strange because i actually didin't touched this file haha, maybe was my IDE, i work with auto-formatting a lot but as my understanding goes it should only affect the current file only and not the whole workspace. However, i just merged dev and left the file as it was before. Really strange issue.

Reply:
Interesting. I'll revert those changes from this PR and will try to investigate a bit further to see how that might happen.